src/me/mrCookieSlime/ExoticGarden/items/GrassSeeds.java

```diff
@@ -6,26 +6,26 @@
 import org.bukkit.block.BlockFace;
 import org.bukkit.inventory.ItemStack;
 
-import me.mrCookieSlime.CSCoreLibPlugin.general.Player.PlayerInventory;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
 import me.mrCookieSlime.Slimefun.Lists.RecipeType;
 import me.mrCookieSlime.Slimefun.Objects.Category;
 import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SimpleSlimefunItem;
 import me.mrCookieSlime.Slimefun.Objects.handlers.ItemInteractionHandler;
-import me.mrCookieSlime.Slimefun.Setup.SlimefunManager;
 
 public class GrassSeeds extends SimpleSlimefunItem<ItemInteractionHandler> {
 
-	public GrassSeeds(Category category, ItemStack item, String id, RecipeType recipeType, ItemStack[] recipe) {
-		super(category, item, id, recipeType, recipe);
+	public GrassSeeds(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
+		super(category, item, recipeType, recipe);
 	}
 	
 	@Override
 	public ItemInteractionHandler getItemHandler() {
 		return (e, p, item) -> {
-			if (SlimefunManager.isItemSimiliar(item, getItem(), true)) {
+			if (isItem(item)) {
 				Block b = e.getClickedBlock();
 				if (b != null && b.getType() == Material.DIRT) {
-					PlayerInventory.consumeItemInHand(p);
+					ItemStack hand = e.getPlayer().getInventory().getItemInMainHand();
+					hand.setAmount(hand.getAmount() - 1);
```

Comment:
.setAmount() can cause a lot of problems sometimes.
It is much safer to use ItemUtils.consume or what it's called from CS-CoreLib2.